controller/tmskillscontroller.go

```diff
@@ -63,13 +62,13 @@ func (c *TMSkillsController) getAllTMSkills() error {
 	for idx := 0; idx < len(tmSkills); idx++ {
 		skillName, err := c.getSkillName(&tmSkills[idx])
 		if err != nil {
-			log.Println("Possible invalid id:", err)
+			c.Println("Possible invalid id:", err)
```

Comment:
Shouldn't this, and the other `Println/Printf`s in this file be `Warnf`s, like you did in `teammemberscontroller.go`?